cmsplugin_articles_ai/models/articles.py

```diff
@@ -106,6 +114,10 @@ class Article(models.Model):
         verbose_name=_("URL slug"),
         db_index=True,
     )
+    language = LanguageField(
```

Comment:
Then we have 2 mechanisms to filter with language. 1. language selection in the article and 2. language filter in views/plugins. That might be more confusing than beneficial, but I'll change it so.